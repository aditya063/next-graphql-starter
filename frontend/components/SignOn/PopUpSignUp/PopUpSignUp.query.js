import { SIGN_UP_MUTATION } from '../../SignUp/SignUp.query';

export { SIGN_UP_MUTATION };
