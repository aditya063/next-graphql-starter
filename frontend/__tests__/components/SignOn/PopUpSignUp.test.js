import React from 'react';
import { shallow } from 'enzyme';
import PopUpSignUp from '../../../components/SignOn/PopUpSignUp/PopUpSignUp';

describe('PopUpSignUp', () => {
  let mockProps;
  let wrapper;

  beforeEach(() => {
    jest.resetAllMocks();
    mockProps = {};
    wrapper = shallow(<PopUpSignUp {...mockProps} />, {
      disableLifecycleMethods: true
    });
  });

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
