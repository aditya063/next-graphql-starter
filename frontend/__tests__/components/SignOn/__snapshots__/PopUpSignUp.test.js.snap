// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopUpSignUp matches snapshot 1`] = `
<Mutation
  errorPolicy="all"
  mutation={
    Object {
      "definitions": Array [
        Object {
          "directives": Array [],
          "kind": "OperationDefinition",
          "name": Object {
            "kind": "Name",
            "value": "SIGN_UP_MUTATION",
          },
          "operation": "mutation",
          "selectionSet": Object {
            "kind": "SelectionSet",
            "selections": Array [
              Object {
                "alias": undefined,
                "arguments": Array [
                  Object {
                    "kind": "Argument",
                    "name": Object {
                      "kind": "Name",
                      "value": "email",
                    },
                    "value": Object {
                      "kind": "Variable",
                      "name": Object {
                        "kind": "Name",
                        "value": "email",
                      },
                    },
                  },
                  Object {
                    "kind": "Argument",
                    "name": Object {
                      "kind": "Name",
                      "value": "password",
                    },
                    "value": Object {
                      "kind": "Variable",
                      "name": Object {
                        "kind": "Name",
                        "value": "password",
                      },
                    },
                  },
                  Object {
                    "kind": "Argument",
                    "name": Object {
                      "kind": "Name",
                      "value": "confirmPassword",
                    },
                    "value": Object {
                      "kind": "Variable",
                      "name": Object {
                        "kind": "Name",
                        "value": "confirmPassword",
                      },
                    },
                  },
                ],
                "directives": Array [],
                "kind": "Field",
                "name": Object {
                  "kind": "Name",
                  "value": "signUp",
                },
                "selectionSet": Object {
                  "kind": "SelectionSet",
                  "selections": Array [
                    Object {
                      "alias": undefined,
                      "arguments": Array [],
                      "directives": Array [],
                      "kind": "Field",
                      "name": Object {
                        "kind": "Name",
                        "value": "id",
                      },
                      "selectionSet": undefined,
                    },
                  ],
                },
              },
            ],
          },
          "variableDefinitions": Array [
            Object {
              "defaultValue": undefined,
              "directives": Array [],
              "kind": "VariableDefinition",
              "type": Object {
                "kind": "NonNullType",
                "type": Object {
                  "kind": "NamedType",
                  "name": Object {
                    "kind": "Name",
                    "value": "String",
                  },
                },
              },
              "variable": Object {
                "kind": "Variable",
                "name": Object {
                  "kind": "Name",
                  "value": "email",
                },
              },
            },
            Object {
              "defaultValue": undefined,
              "directives": Array [],
              "kind": "VariableDefinition",
              "type": Object {
                "kind": "NonNullType",
                "type": Object {
                  "kind": "NamedType",
                  "name": Object {
                    "kind": "Name",
                    "value": "String",
                  },
                },
              },
              "variable": Object {
                "kind": "Variable",
                "name": Object {
                  "kind": "Name",
                  "value": "password",
                },
              },
            },
            Object {
              "defaultValue": undefined,
              "directives": Array [],
              "kind": "VariableDefinition",
              "type": Object {
                "kind": "NonNullType",
                "type": Object {
                  "kind": "NamedType",
                  "name": Object {
                    "kind": "Name",
                    "value": "String",
                  },
                },
              },
              "variable": Object {
                "kind": "Variable",
                "name": Object {
                  "kind": "Name",
                  "value": "confirmPassword",
                },
              },
            },
          ],
        },
      ],
      "kind": "Document",
      "loc": Object {
        "end": 234,
        "start": 0,
      },
    }
  }
  onCompleted={[Function]}
  onError={[Function]}
  variables={
    Object {
      "confirmPassword": "",
      "email": "",
      "password": "",
    }
  }
>
  <Component />
</Mutation>
`;
